% Generated by roxygen2: do not edit by hand
% Please edit documentation in R/zenith_gsa.R
\name{zenith_gsa}
\alias{zenith_gsa}
\alias{zenith_gsa,MArrayLM,GeneSetCollection-method}
\alias{zenith_gsa,MArrayLM,GeneSetCollection,ANY-method}
\title{Perform gene set analysis using zenith}
\usage{
zenith_gsa(
  fit,
  geneSets,
  coefs,
  use.ranks = FALSE,
  n_genes_min = 10,
  inter.gene.cor = 0.01,
  progressbar = TRUE,
  ...
)

\S4method{zenith_gsa}{MArrayLM,GeneSetCollection}(
  fit,
  geneSets,
  coefs,
  use.ranks = FALSE,
  n_genes_min = 10,
  inter.gene.cor = 0.01,
  progressbar = TRUE,
  ...
)
}
\arguments{
\item{fit}{results from \code{dream()}}

\item{geneSets}{\code{GeneSetCollection}}

\item{coefs}{list of coefficients to test using \code{topTable(fit, coef=coefs[[i]])}}

\item{use.ranks}{do a rank-based test \code{TRUE} or a parametric test \code{FALSE}? default: FALSE}

\item{n_genes_min}{minumum number of genes in a geneset}

\item{inter.gene.cor}{if NA, estimate correlation from data.  Otherwise, use specified value}

\item{progressbar}{if TRUE, show progress bar}

\item{...}{other arguments}
}
\value{
\code{data.frame} of results for each gene set and cell type
}
\description{
Perform a competitive gene set analysis accounting for correlation between genes.
}
\details{
This code adapts the widely used \code{camera()} analysis (Wu and Smyth, 2012) in the \code{limma} package (Ritchie, et al. 2015) to the case of linear (mixed) models used by \code{variancePartition::dream()}.
}
\examples{
# Load packages
library(edgeR)
library(variancePartition)
library(tweeDEseqCountData)

# Load RNA-seq data from LCL's
data(pickrell)
geneCounts = exprs(pickrell.eset)
df_metadata = pData(pickrell.eset)

# Filter genes
# Note this is low coverage data, so just use as code example
dsgn = model.matrix(~ gender, df_metadata)
keep = filterByExpr(geneCounts, dsgn, min.count=5)

# Compute library size normalization
dge = DGEList(counts = geneCounts[keep,])
dge = calcNormFactors(dge)

# Estimate precision weights using voom
vobj = voomWithDreamWeights(dge, ~ gender, df_metadata)

# Apply dream analysis
fit = dream(vobj, ~ gender, df_metadata)
fit = eBayes(fit)

# Load Hallmark genes from MSigDB
# use gene 'SYMBOL', or 'ENSEMBL' id
# use get_GeneOntology() to load Gene Ontology
gs = get_MSigDB("H", to="ENSEMBL")
   
# Run zenith analysis
res.gsa = zenith_gsa(fit, gs, 'gendermale', progressbar=FALSE )

# Show top gene sets
head(res.gsa, 2)

# for each cell type select 3 genesets with largest t-statistic
# and 1 geneset with the lowest
# Grey boxes indicate the gene set could not be evaluted because
#    to few genes were represented
plotZenithResults(res.gsa)
}
\references{
Wu, Di, and Gordon K. Smyth. "Camera: a competitive gene set test accounting for inter-gene correlation." Nucleic acids research 40.17 (2012): e133-e133.

Ritchie, M. E., Phipson, B., Wu, D. I., Hu, Y., Law, C. W., Shi, W., & Smyth, G. K. (2015). limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic acids research, 43(7), e47-e47.
}
\seealso{
\code{limma::camera}
}
