<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="description" content="zenith">
<title>Zenith gene set testing after dream analysis â€¢ zenith</title>
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.2.2/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.2.2/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Zenith gene set testing after dream analysis">
<meta property="og:description" content="zenith">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-dark navbar-expand-lg bg-primary"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">zenith</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.5.2</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="active nav-item">
  <a class="nav-link" href="../articles/zenith.html">Get started</a>
</li>
<li class="nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/loading_genesets.html">Loading gene sets</a>
  </div>
</li>
<li class="nav-item">
  <a class="nav-link" href="../news/index.html">Changelog</a>
</li>
      </ul>
<form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off">
</form>

      <ul class="navbar-nav">
<li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/DiseaseNeuroGenomics/zenith/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>

    
  </div>
</nav><div class="container template-article">



<script src="zenith_files/kePrint-0.0.1/kePrint.js"></script><link href="zenith_files/lightable-0.0.1/lightable.css" rel="stylesheet">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Zenith gene set testing after dream analysis</h1>
                        <h4 data-toc-skip class="author"><a href="http://gabrielhoffman.github.io" class="external-link">Gabriel Hoffman</a></h4>
            <address class="author_afil">
      <p>Icahn School of Medicine at Mount Sinai, New
York</p>
<br><small class="dont-index">Source: <a href="https://github.com/DiseaseNeuroGenomics/zenith/blob/HEAD/vignettes/zenith.Rmd" class="external-link"><code>vignettes/zenith.Rmd</code></a></small>
      <div class="d-none name"><code>zenith.Rmd</code></div>
    </address>
</div>

    
    
<p><code>zenith</code> performs gene set analysis on the result of
differential expression using linear (mixed) modeling with <a href="https://doi.org/10.1093/bioinformatics/btaa687" class="external-link">dream</a> by
considering the correlation between gene expression traits. This package
implements the <a href="https://www.rdocumentation.org/packages/limma/versions/3.28.14/topics/camera" class="external-link">camera</a>
method from the <a href="https://bioconductor.org/packages/limma/" class="external-link">limma</a> package
proposed by <a href="https://doi.org/10.1093/nar/gks461" class="external-link">Wu and Smyth
(2012)</a>. <code><a href="../reference/zenith.html">zenith()</a></code> is a simple extension of
<code><a href="https://rdrr.io/pkg/limma/man/camera.html" class="external-link">camera()</a></code> to be compatible with linear (mixed) models
implemented in <code><a href="http://DiseaseNeurogenomics.github.io/variancePartition/reference/dream-method.html" class="external-link">dream()</a></code>.</p>
<div class="section level2">
<h2 id="standard-workflow">Standard workflow<a class="anchor" aria-label="anchor" href="#standard-workflow"></a>
</h2>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># Load packages</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://DiseaseNeuroGenomics.github.io/zenith" class="external-link">zenith</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://bioinf.wehi.edu.au/edgeR/" class="external-link">edgeR</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="http://bioconductor.org/packages/variancePartition" class="external-link">variancePartition</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="http://www.creal.cat/jrgonzalez/software.htm" class="external-link">tweeDEseqCountData</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="http://haozhu233.github.io/kableExtra/" class="external-link">kableExtra</a></span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Load RNA-seq data from LCL's</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="va">pickrell</span><span class="op">)</span></span>
<span><span class="va">geneCounts</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/Biobase/man/exprs.html" class="external-link">exprs</a></span><span class="op">(</span><span class="va">pickrell.eset</span><span class="op">)</span></span>
<span><span class="va">df_metadata</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/Biobase/man/phenoData.html" class="external-link">pData</a></span><span class="op">(</span><span class="va">pickrell.eset</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Filter genes</span></span>
<span><span class="co"># Note this is low coverage data, so just use as code example</span></span>
<span><span class="va">dsgn</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/stats/model.matrix.html" class="external-link">model.matrix</a></span><span class="op">(</span><span class="op">~</span> <span class="va">gender</span>, <span class="va">df_metadata</span><span class="op">)</span></span>
<span><span class="va">keep</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/edgeR/man/filterByExpr.html" class="external-link">filterByExpr</a></span><span class="op">(</span><span class="va">geneCounts</span>, <span class="va">dsgn</span>, min.count<span class="op">=</span><span class="fl">5</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Compute library size normalization</span></span>
<span><span class="va">dge</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/edgeR/man/DGEList.html" class="external-link">DGEList</a></span><span class="op">(</span>counts <span class="op">=</span> <span class="va">geneCounts</span><span class="op">[</span><span class="va">keep</span>,<span class="op">]</span><span class="op">)</span></span>
<span><span class="va">dge</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/edgeR/man/calcNormFactors.html" class="external-link">calcNormFactors</a></span><span class="op">(</span><span class="va">dge</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Estimate precision weights using voom</span></span>
<span><span class="va">vobj</span> <span class="op">=</span> <span class="fu"><a href="http://DiseaseNeurogenomics.github.io/variancePartition/reference/voomWithDreamWeights.html" class="external-link">voomWithDreamWeights</a></span><span class="op">(</span><span class="va">dge</span>, <span class="op">~</span> <span class="va">gender</span>, <span class="va">df_metadata</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Apply dream analysis</span></span>
<span><span class="va">fit</span> <span class="op">=</span> <span class="fu"><a href="http://DiseaseNeurogenomics.github.io/variancePartition/reference/dream-method.html" class="external-link">dream</a></span><span class="op">(</span><span class="va">vobj</span>, <span class="op">~</span> <span class="va">gender</span>, <span class="va">df_metadata</span><span class="op">)</span></span>
<span><span class="va">fit</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/limma/man/ebayes.html" class="external-link">eBayes</a></span><span class="op">(</span><span class="va">fit</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Load get_MSigDB database, Hallmark genes</span></span>
<span><span class="co"># use gene 'SYMBOL', or 'ENSEMBL' id</span></span>
<span><span class="co"># use get_GeneOntology() to load Gene Ontology</span></span>
<span><span class="va">msdb.gs</span> <span class="op">=</span> <span class="fu"><a href="../reference/get_MSigDB.html">get_MSigDB</a></span><span class="op">(</span><span class="st">"H"</span>, to<span class="op">=</span><span class="st">"ENSEMBL"</span><span class="op">)</span></span>
<span>   </span>
<span><span class="co"># Run zenith analysis, and specific which coefficient to evaluate</span></span>
<span><span class="va">res.gsa</span> <span class="op">=</span> <span class="fu"><a href="../reference/zenith_gsa-methods.html">zenith_gsa</a></span><span class="op">(</span><span class="va">fit</span>, <span class="va">msdb.gs</span>, <span class="st">'gendermale'</span>, progressbar<span class="op">=</span><span class="cn">FALSE</span> <span class="op">)</span></span>
<span></span>
<span><span class="co"># Show top gene sets: head(res.gsa)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kable_styling.html" class="external-link">kable_styling</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/knitr/man/kable.html" class="external-link">kable</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">res.gsa</span><span class="op">)</span>, row.names<span class="op">=</span><span class="cn">FALSE</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<table class="table table" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
coef
</th>
<th style="text-align:left;">
Geneset
</th>
<th style="text-align:right;">
NGenes
</th>
<th style="text-align:right;">
Correlation
</th>
<th style="text-align:right;">
delta
</th>
<th style="text-align:right;">
se
</th>
<th style="text-align:right;">
p.less
</th>
<th style="text-align:right;">
p.greater
</th>
<th style="text-align:right;">
PValue
</th>
<th style="text-align:left;">
Direction
</th>
<th style="text-align:right;">
FDR
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
gendermale
</td>
<td style="text-align:left;">
M5890_HALLMARK_TNFA_SIGNALING_VIA_NFKB
</td>
<td style="text-align:right;">
118
</td>
<td style="text-align:right;">
0.01
</td>
<td style="text-align:right;">
-0.9947159
</td>
<td style="text-align:right;">
0.1619433
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:right;">
0.0000001
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:right;">
0.0000024
</td>
</tr>
<tr>
<td style="text-align:left;">
gendermale
</td>
<td style="text-align:left;">
M5892_HALLMARK_CHOLESTEROL_HOMEOSTASIS
</td>
<td style="text-align:right;">
37
</td>
<td style="text-align:right;">
0.01
</td>
<td style="text-align:right;">
-1.0830969
</td>
<td style="text-align:right;">
0.2293248
</td>
<td style="text-align:right;">
0.0000062
</td>
<td style="text-align:right;">
0.9999938
</td>
<td style="text-align:right;">
0.0000123
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:right;">
0.0002892
</td>
</tr>
<tr>
<td style="text-align:left;">
gendermale
</td>
<td style="text-align:left;">
M5932_HALLMARK_INFLAMMATORY_RESPONSE
</td>
<td style="text-align:right;">
93
</td>
<td style="text-align:right;">
0.01
</td>
<td style="text-align:right;">
-0.7885108
</td>
<td style="text-align:right;">
0.1720138
</td>
<td style="text-align:right;">
0.0000102
</td>
<td style="text-align:right;">
0.9999898
</td>
<td style="text-align:right;">
0.0000205
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:right;">
0.0003208
</td>
</tr>
<tr>
<td style="text-align:left;">
gendermale
</td>
<td style="text-align:left;">
M5947_HALLMARK_IL2_STAT5_SIGNALING
</td>
<td style="text-align:right;">
105
</td>
<td style="text-align:right;">
0.01
</td>
<td style="text-align:right;">
-0.6466016
</td>
<td style="text-align:right;">
0.1670231
</td>
<td style="text-align:right;">
0.0001240
</td>
<td style="text-align:right;">
0.9998760
</td>
<td style="text-align:right;">
0.0002481
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:right;">
0.0029152
</td>
</tr>
<tr>
<td style="text-align:left;">
gendermale
</td>
<td style="text-align:left;">
M5930_HALLMARK_EPITHELIAL_MESENCHYMAL_TRANSITION
</td>
<td style="text-align:right;">
53
</td>
<td style="text-align:right;">
0.01
</td>
<td style="text-align:right;">
-0.6132930
</td>
<td style="text-align:right;">
0.2029187
</td>
<td style="text-align:right;">
0.0017754
</td>
<td style="text-align:right;">
0.9982246
</td>
<td style="text-align:right;">
0.0035508
</td>
<td style="text-align:left;">
Down
</td>
<td style="text-align:right;">
0.0307816
</td>
</tr>
<tr>
<td style="text-align:left;">
gendermale
</td>
<td style="text-align:left;">
M5925_HALLMARK_E2F_TARGETS
</td>
<td style="text-align:right;">
109
</td>
<td style="text-align:right;">
0.01
</td>
<td style="text-align:right;">
0.4949455
</td>
<td style="text-align:right;">
0.1656826
</td>
<td style="text-align:right;">
0.9980352
</td>
<td style="text-align:right;">
0.0019648
</td>
<td style="text-align:right;">
0.0039296
</td>
<td style="text-align:left;">
Up
</td>
<td style="text-align:right;">
0.0307816
</td>
</tr>
</tbody>
</table>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># for each cell type select 3 genesets with largest t-statistic</span></span>
<span><span class="co"># and 1 geneset with the lowest</span></span>
<span><span class="co"># Grey boxes indicate the gene set could not be evaluted because</span></span>
<span><span class="co">#    to few genes were represented</span></span>
<span><span class="fu"><a href="../reference/plotZenithResults.html">plotZenithResults</a></span><span class="op">(</span><span class="va">res.gsa</span><span class="op">)</span></span></code></pre></div>
<p><img src="zenith_files/figure-html/plots-1.png" width="700"></p>
</div>
<div class="section level2">
<h2 id="session-info">Session Info<a class="anchor" aria-label="anchor" href="#session-info"></a>
</h2>
<pre><code><span><span class="co">## R version 4.3.0 (2023-04-21)</span></span>
<span><span class="co">## Platform: x86_64-apple-darwin22.4.0 (64-bit)</span></span>
<span><span class="co">## Running under: macOS 14.2.1</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Matrix products: default</span></span>
<span><span class="co">## BLAS:   /Users/gabrielhoffman/prog/R-4.3.0/lib/libRblas.dylib </span></span>
<span><span class="co">## LAPACK: /usr/local/Cellar/r/4.3.0_1/lib/R/lib/libRlapack.dylib;  LAPACK version 3.11.0</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## locale:</span></span>
<span><span class="co">## [1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## time zone: America/New_York</span></span>
<span><span class="co">## tzcode source: internal</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## attached base packages:</span></span>
<span><span class="co">## [1] stats4    stats     graphics  grDevices utils     datasets  methods  </span></span>
<span><span class="co">## [8] base     </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## other attached packages:</span></span>
<span><span class="co">##  [1] org.Hs.eg.db_3.17.0       AnnotationDbi_1.62.1     </span></span>
<span><span class="co">##  [3] IRanges_2.34.1            S4Vectors_0.38.1         </span></span>
<span><span class="co">##  [5] msigdbr_7.5.1             kableExtra_1.3.4         </span></span>
<span><span class="co">##  [7] tweeDEseqCountData_1.38.0 Biobase_2.60.0           </span></span>
<span><span class="co">##  [9] BiocGenerics_0.46.0       variancePartition_1.33.9 </span></span>
<span><span class="co">## [11] BiocParallel_1.34.2       ggplot2_3.4.4            </span></span>
<span><span class="co">## [13] edgeR_3.42.4              zenith_1.5.2             </span></span>
<span><span class="co">## [15] limma_3.56.2              knitr_1.43               </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## loaded via a namespace (and not attached):</span></span>
<span><span class="co">##   [1] rstudioapi_0.15.0           jsonlite_1.8.5             </span></span>
<span><span class="co">##   [3] magrittr_2.0.3              farver_2.1.1               </span></span>
<span><span class="co">##   [5] nloptr_2.0.3                rmarkdown_2.22             </span></span>
<span><span class="co">##   [7] fs_1.6.2                    zlibbioc_1.46.0            </span></span>
<span><span class="co">##   [9] ragg_1.2.5                  vctrs_0.6.3                </span></span>
<span><span class="co">##  [11] memoise_2.0.1               minqa_1.2.5                </span></span>
<span><span class="co">##  [13] RCurl_1.98-1.12             webshot_0.5.4              </span></span>
<span><span class="co">##  [15] htmltools_0.5.5             S4Arrays_1.2.0             </span></span>
<span><span class="co">##  [17] progress_1.2.2              curl_5.0.0                 </span></span>
<span><span class="co">##  [19] broom_1.0.5                 sass_0.4.6                 </span></span>
<span><span class="co">##  [21] KernSmooth_2.23-21          bslib_0.4.2                </span></span>
<span><span class="co">##  [23] desc_1.4.2                  plyr_1.8.8                 </span></span>
<span><span class="co">##  [25] pbkrtest_0.5.2              cachem_1.0.8               </span></span>
<span><span class="co">##  [27] lifecycle_1.0.3             iterators_1.0.14           </span></span>
<span><span class="co">##  [29] pkgconfig_2.0.3             Matrix_1.5-4.1             </span></span>
<span><span class="co">##  [31] R6_2.5.1                    fastmap_1.1.1              </span></span>
<span><span class="co">##  [33] GenomeInfoDbData_1.2.10     rbibutils_2.2.13           </span></span>
<span><span class="co">##  [35] MatrixGenerics_1.12.0       digest_0.6.33              </span></span>
<span><span class="co">##  [37] numDeriv_2016.8-1.1         colorspace_2.1-0           </span></span>
<span><span class="co">##  [39] rprojroot_2.0.3             textshaping_0.3.6          </span></span>
<span><span class="co">##  [41] GenomicRanges_1.52.0        RSQLite_2.3.1              </span></span>
<span><span class="co">##  [43] labeling_0.4.2              filelock_1.0.2             </span></span>
<span><span class="co">##  [45] RcppZiggurat_0.1.6          fansi_1.0.4                </span></span>
<span><span class="co">##  [47] httr_1.4.6                  abind_1.4-5                </span></span>
<span><span class="co">##  [49] compiler_4.3.0              withr_2.5.0                </span></span>
<span><span class="co">##  [51] bit64_4.0.5                 aod_1.3.2                  </span></span>
<span><span class="co">##  [53] backports_1.4.1             DBI_1.1.3                  </span></span>
<span><span class="co">##  [55] highr_0.10                  gplots_3.1.3               </span></span>
<span><span class="co">##  [57] MASS_7.3-60                 DelayedArray_0.26.3        </span></span>
<span><span class="co">##  [59] corpcor_1.6.10              gtools_3.9.4               </span></span>
<span><span class="co">##  [61] caTools_1.18.2              tools_4.3.0                </span></span>
<span><span class="co">##  [63] remaCor_0.0.17              glue_1.6.2                 </span></span>
<span><span class="co">##  [65] nlme_3.1-162                grid_4.3.0                 </span></span>
<span><span class="co">##  [67] reshape2_1.4.4              generics_0.1.3             </span></span>
<span><span class="co">##  [69] gtable_0.3.3                tidyr_1.3.0                </span></span>
<span><span class="co">##  [71] hms_1.1.3                   xml2_1.3.4                 </span></span>
<span><span class="co">##  [73] utf8_1.2.3                  XVector_0.40.0             </span></span>
<span><span class="co">##  [75] pillar_1.9.0                stringr_1.5.0              </span></span>
<span><span class="co">##  [77] babelgene_22.9              splines_4.3.0              </span></span>
<span><span class="co">##  [79] dplyr_1.1.2                 BiocFileCache_2.8.0        </span></span>
<span><span class="co">##  [81] lattice_0.21-8              bit_4.0.5                  </span></span>
<span><span class="co">##  [83] annotate_1.78.0             tidyselect_1.2.0           </span></span>
<span><span class="co">##  [85] locfit_1.5-9.7              Biostrings_2.68.1          </span></span>
<span><span class="co">##  [87] SummarizedExperiment_1.30.1 svglite_2.1.1              </span></span>
<span><span class="co">##  [89] RhpcBLASctl_0.23-42         xfun_0.39                  </span></span>
<span><span class="co">##  [91] matrixStats_1.0.0           KEGGgraph_1.60.0           </span></span>
<span><span class="co">##  [93] stringi_1.7.12              yaml_2.3.7                 </span></span>
<span><span class="co">##  [95] boot_1.3-28.1               evaluate_0.21              </span></span>
<span><span class="co">##  [97] codetools_0.2-19            tibble_3.2.1               </span></span>
<span><span class="co">##  [99] Rgraphviz_2.44.0            graph_1.78.0               </span></span>
<span><span class="co">## [101] cli_3.6.1                   xtable_1.8-4               </span></span>
<span><span class="co">## [103] systemfonts_1.0.4           Rdpack_2.4                 </span></span>
<span><span class="co">## [105] munsell_0.5.0               jquerylib_0.1.4            </span></span>
<span><span class="co">## [107] Rcpp_1.0.11                 GenomeInfoDb_1.36.1        </span></span>
<span><span class="co">## [109] EnvStats_2.7.0              dbplyr_2.3.2               </span></span>
<span><span class="co">## [111] png_0.1-8                   Rfast_2.0.7                </span></span>
<span><span class="co">## [113] XML_3.99-0.14               parallel_4.3.0             </span></span>
<span><span class="co">## [115] pkgdown_2.0.7               blob_1.2.4                 </span></span>
<span><span class="co">## [117] prettyunits_1.1.1           bitops_1.0-7               </span></span>
<span><span class="co">## [119] lme4_1.1-34                 viridisLite_0.4.2          </span></span>
<span><span class="co">## [121] mvtnorm_1.2-2               GSEABase_1.62.0            </span></span>
<span><span class="co">## [123] lmerTest_3.1-3              scales_1.2.1               </span></span>
<span><span class="co">## [125] purrr_1.0.2                 crayon_1.5.2               </span></span>
<span><span class="co">## [127] fANCOVA_0.6-1               rlang_1.1.1                </span></span>
<span><span class="co">## [129] rvest_1.0.3                 EnrichmentBrowser_2.30.1   </span></span>
<span><span class="co">## [131] KEGGREST_1.40.0</span></span></code></pre>
</div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p></p>
<p>Developed by <a href="http://gabrielhoffman.github.io" class="external-link">Gabriel Hoffman</a>.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

    </footer>
</div>

  

  

  </body>
</html>
